spark/arctern_spark/geodataframe.py

```diff
@@ -71,7 +120,10 @@ def _set_geometries(self, cols, crs=None):
 
     def set_geometry(self, col, inplace=False, crs=None):
         """
-        Sets an existing column in the GeoDataFrame to a geometry column, which is used to perform geometric calculations later.
+        Set an existing column in the GeoDataFrame to a geometry column, which is used to perform geometric calculations later.
+
+        Setting an column to a geometry column will attemp to construct geometry for each row of this column,
```

Comment:
This function attempts to construct geometries for each row of the given column, so all data in \`\`col\`\` should be in WKT or WKB format.